router/batchrouter/batchrouter.go

```diff
@@ -5,7 +5,7 @@ import jsoniter "github.com/json-iterator/go"
 var (
 	json                    = jsoniter.ConfigCompatibleWithStandardLibrary
 	objectStoreDestinations = []string{"S3", "GCS", "AZURE_BLOB", "MINIO", "DIGITAL_OCEAN_SPACES"}
-	asyncDestinations       = []string{"MARKETO_BULK_UPLOAD", "BINGADS_AUDIENCE", "ELOQUA"}
+	asyncDestinations       = []string{"MARKETO_BULK_UPLOAD", "BINGADS_AUDIENCE", "ELOQUA", "YANDEX_METRICA_OFFLINE_EVENTS"}
```

Comment:
we can use the `var AsyncDestinations` defined in `router/batchrouter/asyncdestinationmanager/common/common.go`. Safer imo